CONTRIBUTING.md

```diff
@@ -173,3 +173,40 @@ Before merging a PR make sure:
 
 Merge the PR by using "Squash and merge" option on Github. Avoid creating merge commits.
 After the merge make sure referenced issues were closed.
+
+### Developing guidelines
+
+#### Building [OCI Images](https://github.com/opencontainers/image-spec/blob/master/spec.md) for multiple arch (linux/arm64, linux/amd64)
```

Comment:
nit: can you keep the title without a link, and add the link to "OCI images" in the line 181?